Tests/test.d

```diff
@@ -41,12 +41,35 @@ void main() {
 	debug {
     Stdout("this is from within debug { }").newline;
   }
-  
+
   foreach_reverse(i,c; name) {
     Stdout.format("{}:{}, ", i, c);
   }
   Stdout.newline;
-  
+
   auto mc = new MyClass(14);
   mc.run();
 }
+
+// "if" in "uniform" should not be parsed as keyword.control.conditional.d
+import std.random : Random, unpredictableSeed, uniform;
+
+// "static" and "if" should be parsed as keyword.control.conditional.d
+// "static" and "if be parsed as two separate words
+static if (true) {}
+
+void main()
+{
+    // "if" should be parsed as keyword.control.conditional.d
+    if (true) {}
+
+// "else" and "if" should be parsed as keyword.control.conditional.d
+// "else" and "if" should parsed as two separate words
+else if (true) {}
```

Comment:
`should parsed` => `should be parsed` ?
Also, would it make sense to have a test that test for 'else if' on a separate line ?